veo que usan try/catch, saben que es? 
tambien en la linea numero [4](https://github.com/danielcabreraufm/contact_manager/blob/master/CONTACT_MANAGER.py#L4) importan `requests.exceptions import HTTPError` entienden que es?

por el momento solo esos comentarios
